From 9147ea0a71a1587a212bee0da9fd20f0668ed67e Mon Sep 17 00:00:00 2001
From: Try Omarchy Release <actions@users.noreply.github.com>
Date: Fri, 25 Sep 2026 03:15:06 +0000
Subject: [PATCH] Refresh the guest package lock

fcitx5 5.1.22-1 -> 5.1.23-1
fcitx5-qt 5.1.15-1 -> 5.1.16-1
parted 3.7-1 -> 3.8-1
systemd 261.3-1 -> 262-1
systemd-libs 261.3-1 -> 262-1
systemd-resolvconf 261.3-1 -> 262-1
systemd-sysvcompat 261.3-1 -> 262-1
---
 guest/packages.lock.json | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/guest/packages.lock.json b/guest/packages.lock.json
index 073fad7..329f90c 100644
--- a/guest/packages.lock.json
+++ b/guest/packages.lock.json
@@ -73,9 +73,9 @@
     "fakeroot": "1:1.37.2-3",
     "fastfetch": "2.68.1-1",
     "fcft": "3.3.3-1",
-    "fcitx5": "5.1.22-1",
+    "fcitx5": "5.1.23-1",
     "fcitx5-gtk": "5.1.7-1",
-    "fcitx5-qt": "5.1.15-1",
+    "fcitx5-qt": "5.1.16-1",
     "fd": "10.5.0-3",
     "ffmpeg": "2:9.0.2-1",
     "fftw": "3.3.11-1",
@@ -492,7 +492,7 @@
     "pambase": "20260616-1",
     "pamixer": "1.6-4",
     "pango": "1:1.58.2-1",
-    "parted": "3.7-1",
+    "parted": "3.8-1",
     "patch": "2.8-1",
     "pciutils": "3.15.0-1",
     "pcre": "8.45-5",
@@ -568,10 +568,10 @@
     "starship": "1.26.0-1",
     "sudo": "1.9.17.p2-6",
     "svt-av1": "4.2.0-1",
-    "systemd": "261.3-1",
-    "systemd-libs": "261.3-1",
-    "systemd-resolvconf": "261.3-1",
-    "systemd-sysvcompat": "261.3-1",
+    "systemd": "262-1",
+    "systemd-libs": "262-1",
+    "systemd-resolvconf": "262-1",
+    "systemd-sysvcompat": "262-1",
     "tar": "1.35-5",
     "tensaku": "0.29.0-1",
     "tesseract": "5.5.3-1",
-- 
2.55.0

